examples/plot_visualize_bad_epochs.py

```diff
@@ -41,41 +42,25 @@
 tmin, tmax = -0.2, 0.8
 events_id = {'famous/first': 5, 'famous/immediate': 6, 'famous/long': 7}
 
-# %%
-# Let us now load all the epochs into memory and concatenate them
-
-import mne  # noqa
-
-epochs = list()
-for run in range(3, 7):
```

Comment:
this is great! But unfortunately, the data is already clean. I think we should pick a run which has at least a few bad segments and shows that it has been fixed